juriscraper/pacer/attachment_page.py

```diff
@@ -265,11 +270,33 @@ def _get_description_from_tr(self, row):
         description = description_text_nodes[0].strip()
         return force_unicode(description)
 
+    @staticmethod
+    def _get_page_count_from_tr_input_value(tr):
+        """Take a row from the attachment table and return the page count as an
+        int extracted from the input value.
+        """
+        try:
+            input = tr.xpath(".//input")[0]
+        except IndexError:
+            return None
+        else:
+            # initial value string "23515655-90555-2"
+            # "90555" is size in bytes "2" is pages
+            value = input.xpath("./@value")[0]
+            split_value = value.split("-")
+            if len(split_value) != 3:
+                return None
+            # after splitting we get our page count value "2"
+            return int(split_value[2])
```

Comment:
This is repeated four times, more or less. Can we generalize it, please?

Reply:
refactored input value field parsing